nestauk/asf_heat_pump_suitability
# 57 add building conservation areas wales

Fixes #57 

# Description

Add building conservation area data for Wales to `add_features.py` pipeline so that 'in_conservation_area' flag on EPC dataset will flag if EPC property is located in a building conservation area in England or Wales.

Summary of changes:
- Add new getter to `get_datasets.py` for Wales building conservation area data
- Add source of Wales data to `config/base.yaml` and `config/README`
- Update `prepare_features/conservation_areas.py` to load and transform Wales building cons area data as well as England data
- Minor changes to logging and notation in `run_scripts/add_features.py`

# Instructions for Reviewer

To test the code run:
`python -i asf_heat_pump_suitability/pipeline/run_scripts/add_features.py --epc_path s3://asf-heat-pump-suitability/outputs/2023_Q2_EPC_enhanced.parquet --save_output s3://asf-heat-pump-suitability/outputs/test.parquet`

Please pay special attention to ...
- Please could you double check I've used the correct dataset ([source](https://datamap.gov.wales/layers/inspire-wg:conservation_areas)). The most recent version I could find was from Sept 2022. This dataset also has a relatively low number of cons areas - around 500.
- Please could you let me know if there are any obvious speed improvements you can see
- Please could you check for general bugs and mistakes

# Checklist:
